src/video.cpp

```diff
@@ -716,12 +722,13 @@ namespace video {
       // Common options
       {
         { "filler_data"s, false },
+        { "forced_idr"s, 1 },
+        { "latency"s, 1 },
+        { "skip_frame"s, 0 },
         { "log_to_dbg"s, []() { return config::sunshine.min_log_level < 2 ? 1 : 0; } },
         { "gops_per_idr"s, 1 },
         { "header_insertion_mode"s, "idr"s },
         { "preencode"s, &config::video.amd.amd_preanalysis },
-        { "qmax"s, 51 },
-        { "qmin"s, 0 },
```

Comment:
This was fixed properly upstream in https://github.com/FFmpeg/FFmpeg/commit/1a549efefb590a83b2516dfd287dbda4e8d6b846